pkgs/backend/sdk/vnscope/classify.py

```
@@ -261,3 +263,137 @@ def plot_heatmap_with_candlestick(
+
+            # next centers according price
+            mapping = sorted(
+                [i for i in range(0, len(centers))],
+                key=lambda i: levels[centers[i]],
+            )
+            blocks = [
+                (i, levels[begins[i]], levels[centers[i]], levels[ends[i]])
+                for i in mapping
+            ]
+
+            for (p, (i, begin, center, end)) in enumerate(blocks):
```

Ruff (0.14.3)

Loop control variable `center` not used within loop body

Rename unused `center` to `_center`

(B007)